specs/waku/v2/waku-filter.md

```diff
@@ -1,28 +1,34 @@
 ---
 title: Waku
-version: 2.0.0-alpha5
-status: Raw
-authors: Oskar Thorén <oskar@status.im>
+version: 2.0.0-beta1
+status: Draft
+authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
 ---
 
 # Table of Contents
 
-TODO
+- [Abstract](#abstract)
+- [Content filtering](#content-filtering)
+  * [Rationale](#rationale)
+  * [Protobuf](#protobuf)
+- [Changelog](#changelog)
+- [Copyright](#copyright)
+- [References](#references)
 
 # Abstract
 
-Filter spec.
+`WakuFilter` is a protocol that enables subscribing to messages that a peer receives. This is a more lightweight version of `WakuRelay` specifically designed for bandwidth restricted devices. This is due to the fact that light nodes subscribe to full-nodes and only receive the messages they desire.
```

Comment:
honestly I don't think this addresses the feedback I gave before, it still isn't very clear what this is or how it works. But let's move on...